src/solve.jl

```diff
@@ -184,7 +184,7 @@ function solve!(system::InitialValueProblem{<:HybridSystem,
             # TODO temporary hack, to be resolved in #447
             options_copy[:mode] = "reach"
         end
-        reach_tube = solve!(ContinuousSystem(loc.A, X0.X, loc.U),
+        reach_tube = solve!(IVP(CLCCS(loc.A, Matrix(1.0I, size(loc.A)), nothing, ConstantInput(loc.U)), X0.X),
```

Comment:
Can't you just do `IVP(loc, X0.X)`? This would resolve #491.

Reply:
Very good! Together with this change i had to update the hybrid tests to use type inputs in the way expected by the algorithm that is to wrap the inputs around a `ConstantInput` or a `VaryingInput` constructor:

```julia
    m1 = ConstrainedLinearControlContinuousSystem(
            A, Matrix(1.0I, size(B, 1), size(B, 1)), inv, ConstantInput(B*U))
```

Eventually (with [MathematicalSystems#76](https://github.com/JuliaReach/MathematicalSystems.jl/issues/76)) this can be given with a notation closer to mathematical writing:

```julia
m1  = @system x' = Ax + Bu, x ∈ inv, u ∈ U
``